bears/vcs/git/GitCommitBear.py

```diff
@@ -109,6 +113,37 @@ def check_shortlog(self,
                 yield Result(
                     self,
                     "Shortlog of HEAD commit does not match given regex.")
+        if shortlog_imperative_check:
```

Comment:
Sure. I'll push the updates once you are done reviewing. :D
ᐧ

On Mon, Apr 4, 2016 at 11:09 PM, Mischa Krüger <notifications@github.com>
wrote:

> In bears/vcs/git/GitCommitBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/315#discussion_r58415472>
> :
>
> > @@ -109,6 +113,37 @@ def check_shortlog(self,
> >                  yield Result(
> >                      self,
> >                      "Shortlog of HEAD commit does not match given regex.")
> > +        if shortlog_imperative_check:
>
> please put a newline before this line :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/315/files/44ad0a60bd0ebd1a734c14f78a7a669e30537549#r58415472>
>